example.sql

```diff
@@ -1,18 +1,34 @@
 \c nc_games_test
 
-      SELECT 
-      reviews.title,
-  reviews.designer,
-  reviews.owner,
-  reviews.review_img_url,
-  reviews.category,
-  reviews.created_at,
-  reviews.votes,
-  reviews.review_id,
-COUNT(comments.comment_id) AS comment_count
---   CAST(reviews.comment_count AS INT)
-      FROM reviews
-      LEFT JOIN comments
-      ON reviews.review_id = comments.review_id
-      GROUP BY reviews.review_id
-      ORDER BY created_at DESC;
\ No newline at end of file
+--       SELECT 
```

Comment:
This file should have been added to your .gitignore to make sure it isn't pushed up to GitHub